scripts/build.py

```diff
@@ -152,9 +153,12 @@ def generate_app_yaml(is_prod_mode=False):
         is_prod_mode: bool. Whether the script is being called
         with --prod_env flag or not.
     """
+    APP_YAML_LOCAL_FILEPATH = APP_DEV_YAML_FILEPATH
+    if is_prod_mode:
+        APP_YAML_LOCAL_FILEPATH = APP_PROD_YAML_FILEPATH
     prod_file_prefix = 'build/'
     content = '# THIS FILE IS AUTOGENERATED, DO NOT MODIFY\n'
-    with python_utils.open_file(APP_DEV_YAML_FILEPATH, 'r') as yaml_file:
+    with python_utils.open_file(APP_YAML_LOCAL_FILEPATH, 'r') as yaml_file:
         content += yaml_file.read()
     for file_path in FILEPATHS_NOT_TO_RENAME:
```

Comment:
Here the `build/` prefix is added for the generated prod file, since we now have _prod_app.yaml_ we can get rid of this code and add `build/` in front of the `webpack_bundles/` in _prod_app.yaml_.

Reply:
Similarly
```python
if not is_prod_mode:
    content = content.replace('version: default', '')
```
can also be removed.

Reply:
Made more updates. This code is removed now, PTAL.